hil/test_common.py

```diff
@@ -30,6 +30,10 @@
 import sys
 import os.path
 import logging
+import re
+
+uuid_pattern = re.compile(
+            "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
```

Comment:
We should also use this definition in the migration tests `tests/unit/migrations.py`, which currently define an equivalent regex in-line.